irohad/model/converters/pb_query_response_factory.hpp

```diff
@@ -20,6 +20,7 @@
 
 #include <responses.pb.h>
 #include <model/account_asset.hpp>
```

Comment:
Fix these two headers as well plz

Reply:
Same issue, this code should be deleted soon.